WebClient/src/main/resources/static/index.html

```diff
@@ -2,10 +2,10 @@
 <html>
 <head>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <script src="/view/ViewMaster.js"></script>
-    <script src="/view/RequestMaster.js"></script>
-    <script src="/view/Executable.js"></script>
-    <script src="/view/EBooksCategory.js"></script>
+    <script src="/js/ViewMaster.js"></script>
+    <script src="/js/RequestMaster.js"></script>
```

Comment:
Let me leave this please. :P Please, I want this names. please.... :D maybe later I will change.